cups/encode.c

```diff
@@ -136,6 +136,7 @@ static const _ipp_option_t ipp_options[] =
   { 0, "fit-to-page-default",	IPP_TAG_BOOLEAN,	IPP_TAG_PRINTER },
   { 0, "fitplot",		IPP_TAG_BOOLEAN,	IPP_TAG_JOB },
   { 0, "fitplot-default",	IPP_TAG_BOOLEAN,	IPP_TAG_PRINTER },
+  { 0, "force-raster",		IPP_TAG_BOOLEAN,	IPP_TAG_JOB },
```

Comment:
I would prefer the name to be "print-as-raster", with a corresponding "print-as-raster-default" boolean attribute in the `IPP_TAG_PRINTER` group.